src/main/java/ua/danit/final_project/repositories/TaskRepository.java

```diff
@@ -2,8 +2,21 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import ua.danit.final_project.entities.Location;
 import ua.danit.final_project.entities.Task;
 
+import java.util.Arrays;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Arrays'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=109&issues=AWXJ1wpi_eEZPtUmsTSs&open=AWXJ1wpi_eEZPtUmsTSs)

[ISSUE_KEY]: AWXJ1wpi_eEZPtUmsTSs